packages/server/utils/predicates.ts

```diff
@@ -1,2 +1,5 @@
 export const isNotNull = <T>(item: T | null | undefined): item is T =>
   item !== null && item !== undefined
+
+export const isNeitherNullNorError = <T>(item: T | Error | null | undefined): item is T =>
```

Comment:
-1 no preference on which helper we choose, but this functionality already exists here: https://github.com/ParabolInc/parabol/blob/9c27f5716901081c321d69ea6f56fb4756a83c1d/packages/server/graphql/isValid.ts

would like to only have 1 instance of this in the codebase